api/app/services/feedback_service.py

```
@@ -222,55 +237,67 @@ async def update_feedback_entry(self, message_id: str,
+                    ) as locked_file_handle:
                         # Note: portalocker.Lock opens the file, so we use its handle or reopen carefully.
-                        # For simplicity and to maintain existing open logic, we will re-open 
+                        # For simplicity and to maintain existing open logic, we will re-open
                         # after lock acquisition, ensuring portalocker primarily handles the lock file mechanism.
                         # A more integrated approach would use locked_file_handle directly if its mode matches.
                         # However, the temp file pattern means we mostly need the lock for the duration of this block.
                         # Re-opening inside the lock context after portalocker creates/validates the lock file:
-                        with open(file_path, 'r') as original, open(temp_path, 'w') as temp:
+                        with open(file_path, "r") as original, open(
+                            temp_path, "w"
+                        ) as temp:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)